Client/YooChooseNotifier.php

```diff
@@ -408,8 +408,10 @@ private function notifyGuzzle6(array $events)
 
         $promise->wait();
 
-        $promise->then(function (Response $response) {
-            $this->log(sprintf('Got asynchronously %s from YooChoose notification POST', $response->getStatusCode()), 'debug');
+        $promise->then(function (ResponseInterface $response) {
+            if (isset($this->logger)) {
+                $this->logger->log(sprintf('Got asynchronously %s from YooChoose notification POST', $response->getStatusCode()), 'debug');
```

Comment:
The arguments for the `log` methods are the other way around. Why not use `debug` method instead of `log`?